concourse_ci/README.md

```diff
@@ -0,0 +1,33 @@
+## Overview
+
+Capture metrics emitted by concourse-ci.
+
+## Setup
+
+Concourse-ci comes bundled with a Datadog metrics emitter. Configure the `Datadog Metric Emitter` by
+setting the Datadog agent host, port and set the prefix to `concourse.ci.`.
+
+```
+Metric Emitter (Datadog):
+    --datadog-agent-host=       Datadog agent host to expose dogstatsd metrics [$CONCOURSE_DATADOG_AGENT_HOST]
+    --datadog-agent-port=       Datadog agent port to expose dogstatsd metrics [$CONCOURSE_DATADOG_AGENT_PORT]
+    --datadog-prefix=           Prefix for all metrics to easily find them in Datadog [$CONCOURSE_DATADOG_PREFIX]
+```
+
```

Comment:
I think this part lacks of formatting. Other integration tiles generally include some subtitles (Installation, Configuration, ...).

Reply:
I thought I would keep it short since the metric emitter configuration is done on the concourse/ATC side. Its worth noting that a prerequisite is to have a working agent somewhere to send metrics to dogstatsd.